litex/build/xilinx/ise.py

```diff
@@ -207,7 +218,8 @@ def build(self, platform, fragment, build_dir="build", build_name="top",
                     _build_xst_files(platform.device, platform.sources, platform.verilog_include_paths, build_name, self.xst_opt)
                     isemode = mode
                 else:
-                    _run_yosys(platform.device, platform.sources, platform.verilog_include_paths, build_name)
+                    if run:
```

Comment:
Where does `run` come from?

Reply:
Oh wait, it is an argument....

Reply:
Right. This is added since it executes yosys right now even when you do not build but load on board

Reply:
Great!